### PR TITLE
Account create mnemonic hint hotfix

### PR DESCRIPTION
**Pull request by maciejnems**
This hotfix is for now, as a lot of people ask how they can restore an account and ask our moderators about that. This is aimed to answer those questions.
Before:
image
After:
image

### CODE REVIEW

File: packages/apps/public/locales/ar/translation.json [hunk 1]
```diff
@@ -674,7 +674,7 @@
   "The resulting signature of the input data, as done with the crypto algorithm from the account. (This could be non-deterministic for some types such as sr25519).": ".(التوقيع الناتج لبيانات الإدخال، كما تم مع خوارزمية التشفير من الحساب. (قد يكون هذا غير حتمي لبعض الأنواع مثل إس أر25519",
   "The scheduling preference for this parachain.": ".تفضيلات الجدولة لهذا  لهذه الكتلة الموازية",
   "The scheduling setting for this parachain.": ".إعداد الجدولة  لهذه الكتلة الموازية",
-  "The secret seed value for this account. Ensure that you keep this in a safe place, with access to the seed you can re-create the account.": ".قيمة البذرة السرية لهذا الحساب.  تأكد من الاحتفاظ بهذا في مكان آمن، مع الوصول إلى البذرة يمكنك إعادة إنشاء الحساب",
+  "The secret seed value for this account. Ensure that you keep this in a safe place, with access to the seed you can re-create the account. If you wish to retrieve your account now, paste your mnemonic seed in the input field on your left and proceed with on-screen instructions. Your account will be recovered.": ".قيمة البذرة السرية لهذا الحساب.  تأكد من الاحتفاظ بهذا في مكان آمن، مع الوصول إلى البذرة يمكنك إعادة إنشاء الحساب",
```
Comment (timorl) [inline packages/apps/public/locales/ar/translation.json:677]: Ouch, it's annoying that we are leaving the translations unchanged. :/

Comment (maciejnems) [reply packages/apps/public/locales/ar/translation.json:677]: Unfortunately I don't know arabic :// polkadot does a lot of similar stuff. Not everything has full translation. I can ask Antek to translate it for us somehow

Comment (michalseweryn) [reply packages/apps/public/locales/ar/translation.json:677]: I think we could leave it only in English for now, but make a ticket for it, and update it as soon as a technical writer can translate any of those.

**Review by michalseweryn — APPROVED** [review]

**Review by timorl — APPROVED** [review]
Reasonable as a hotfix.

**Pull request merged**